src/service/viewer-impl.js

```diff
@@ -206,6 +212,21 @@ export class Viewer {
         log.fine(TAG_, 'replace url:' + this.win.location.href);
       }
     }
+
+    // Check if by the time the `Viewer`
+    // instance is constructed, the document is already `visible`.
+    this.onVisibilityChange_();
```

Comment:
There's still one instance of `visibilityObservable_.fire()` that's not covered. Please see the `receiveMessage `.